src/ui/misc/FragmentSafeAreaView.tsx

```diff
@@ -3,10 +3,18 @@ import { SafeAreaView, ViewProps } from 'react-native';
 import { ComponentColors } from '../../styles';
 import { StatusBarView } from '../../components/StatusBarView';
 
-export const FragmentSafeAreaViewWithoutTabBar = (props: ViewProps & { children: any }) => (
+interface OwnProps {
+    children: React.ReactNode | React.ReactNode[];
+    topBackgroundColor?: string;
+    bottomBackgroundColor?: string;
+}
+
+type Props = ViewProps & OwnProps;
```

Comment:
Why `ViewProps` is needed here?

Reply:
I did not check if it was used, it can be removed.